app/main.py

```diff
@@ -1,6 +1,24 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def wrapper(*args, **kwargs) -> Any:
+        func_name = func.__name__
+        key_args = args
```

Comment:
current implementation only considers positional arguments as keys for caching, without keyword arguments 
in order to include both args and kwargs in the key you should use
`key = args + tuple(kwargs.items())` instead of `key_args = args`